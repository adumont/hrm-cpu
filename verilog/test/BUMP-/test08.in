42 fd
18 46
21 2a
40 05
27 1a
36 bd
02 65
27 6b
15 0c
53 f8
11 a3
56 8c
23 97
19 bc
06 97
52 a7
48 30
27 62
57 42
33 f0
42 b1
51 69
28 f3
41 24
08 f4
58 0f
52 11
22 c3
28 c2
11 05
57 93
10 a4
47 aa
49 5d
46 d3
51 2b
02 31
42 7f
59 0a
17 ad
19 30
01 e1
47 e1
19 d5
34 3c
40 2a
