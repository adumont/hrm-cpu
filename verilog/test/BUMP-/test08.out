fc
45
29
04
19
bc
64
6a
0b
f7
a2
8b
96
bb
96
a6
2f
61
41
ef
b0
68
f2
23
f3
0e
10
c2
c1
04
92
a3
a9
5c
d2
2a
30
7e
09
ac
2f
e0
e0
d4
3b
29
